source/MaterialXRuntime/RtFileIo.cpp

```diff
@@ -1489,6 +1489,10 @@ void RtFileIo::read(const FilePath& documentPath, const FileSearchPath& searchPa
 
         PvtStage* stage = PvtStage::cast(_stage.get());
         readDocument(document, stage, options);
+
+        PvtApi* api = PvtApi::cast(RtApi::get());
```

Comment:
Nitpick. Do you mind turning this into a local function and describe what it does ?
Have a feeling we'll forget over time. And both must be called to do the correct thing.